utils.py

```diff
@@ -33,51 +33,53 @@ def poly2mask(vertex_row_coords, vertex_col_coords, shape):
             for within the polygon, False for outside the polygon
     """
     fill_row_coords, fill_col_coords = draw.polygon(
-        vertex_row_coords,
-        vertex_col_coords,
-        shape
+        vertex_row_coords, vertex_col_coords, shape
     )
     mask = np.zeros(shape, dtype=np.bool)
     mask[fill_row_coords, fill_col_coords] = True
     return mask
 
 
-def get_points(data, img_path, roi_points, shape):
+def get_points(data, img_path, roi_points, reactOHIF=True):
+
     """Convert x,y point data into 3D masks.
 
     Args:
-        data (3D np array) the result
-        roi_image_path (string) provides orientation and slice info
-        roi_points (dict) the part of the roi dictionary that holds the point data
+        data (3D np array): The result
+        roi_image_path (string): Provides orientation and slice info
+        roi_points (dict): The part of the roi dictionary that holds the point data
```

Comment:
Not actually an argument to get_points()?

Reply:
roi_image_path -> image_path